cmd/crossplane/main.go

```diff
@@ -84,6 +94,13 @@ func main() {
 	// logger is *very* verbose even at info level. This is not really needed,
 	// but otherwise we get a warning from the controller-runtime.
 	ctrl.SetLogger(zap.New(zap.WriteTo(io.Discard)))
+
+	fs := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
+	klog.InitFlags(fs)
+	fs.Parse([]string{"--v=3"}) // nolint:errcheck
+	klogr := logr.New(&logrFilter{zl.GetSink()})
+	klog.SetLogger(klogr)
```

Comment:
Ditto here WRT adding an explanatory comment.

It looks a bit like we're discarding `v=4` and lower, and sending `v=3` and lower to our zap logger? Is that right?

Reply:
Yes.

Reply:
done